experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -60,33 +109,25 @@ export class PrometheusExporter extends MetricReader {
       aggregationTemporalitySelector: _instrumentType =>
         AggregationTemporality.CUMULATIVE,
     });
-    this._host =
-      config.host ||
-      process.env.OTEL_EXPORTER_PROMETHEUS_HOST ||
-      PrometheusExporter.DEFAULT_OPTIONS.host;
-    this._port =
-      config.port ||
-      Number(process.env.OTEL_EXPORTER_PROMETHEUS_PORT) ||
-      PrometheusExporter.DEFAULT_OPTIONS.port;
     this._prefix = config.prefix || PrometheusExporter.DEFAULT_OPTIONS.prefix;
     this._appendTimestamp =
       typeof config.appendTimestamp === 'boolean'
         ? config.appendTimestamp
         : PrometheusExporter.DEFAULT_OPTIONS.appendTimestamp;
     // unref to prevent prometheus exporter from holding the process open on exit
-    this._server = createServer(this._requestHandler).unref();
+    this._server = prepareServer(config, this._requestHandler).unref();
```

Comment:
@Think we should be able to move only call this when preventServerStart is not false. Also the main use case for setting it to true is to avoid creating a second server socket and use an existing server by calling request handler function `getMetricsRequestHandler`. Potentially avoids unused resources